src/wp-includes/html-api/class-wp-html-active-formatting-elements.php

```diff
@@ -86,6 +86,22 @@ public function current_node() {
 		return $current_node ? $current_node : null;
 	}
 
+	/**
+	 * Inserts a "marker" at the end of the list of active formatting elements.
+	 *
+	 * > The markers are inserted when entering applet, object, marquee,
+	 * > template, td, th, and caption elements, and are used to prevent
+	 * > formatting from "leaking" into applet, object, marquee, template,
+	 * > td, th, and caption elements.
+	 *
+	 * @see https://html.spec.whatwg.org/#concept-parser-marker
+	 *
+	 * @since 6.7.0
+	 */
+	public function insert_marker() {
```

Comment:
```suggestion
	public function insert_marker(): void {
```

Reply:
The `void` return type hint was introduced in PHP 7.1, and WP still supports PHP 7.0, so I don't think this can be added.

Reply:
I believe that's changed. [6.6 dropped support for PHP 7.0 and 7.1](https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/):

> In WordPress 6.6:
[Dropped support](https://make.wordpress.org/core/2024/04/08/dropping-support-for-php-7-1/) for PHP 7.0 & 7.1.

Reply:
Ah! Your good 😄

Reply:
The [requirements page](https://wordpress.org/about/requirements/) needs to be updated.

Reply:
Good spot, working on getting it fixed: https://meta.trac.wordpress.org/ticket/7723